test/test_element_abs.f90

```diff
@@ -20,7 +20,7 @@ program test_element_abs
 
 
   do i = 1, 100
-    y0 = i
+    y0(i) = i
```

Comment:
Wouldn't it be better to include negative numbers in a test of `abs`?

Reply:
Yep - this usertype goes into a function first that multiplies it by -2. Then it's checked, so we are checking negative numbers.

Reply:
What I meant was: maybe include both positive and negative numbers? :)

Reply:
Ah, you know what, this is good enough.